test/dynamo/test_dynamo_integrations_util.py

```diff
@@ -84,6 +84,18 @@ def test_get_graph_hash(self):
     xla_out_2 = xla_dummy_model(xla_input)
     assert (hash == torch_xla._XLAC._get_graph_hash([xla_out_2]))
 
+  def test_get_graph_hash_raises_error_on_non_xla_tensor(self):
+    tensors = [
+        torch.rand(10, device=torch_xla.device()),
+        torch.rand(10),
+        torch.rand(10, device=torch_xla.device()),
+    ]
+    error_message = (
+        "Expected all tensors in the given list to be XLA tensors. "
+        "Element at index 1 is not an XLA tensor. Got: torch.FloatTensor")
+    with self.assertRaisesRegex(RuntimeError, expected_regex=error_message):
```

Comment:
Can we do a straight string comparison instead of regex match? We don't want `.` to be interpreted as a regex meta character, for example. Also, I suspect that assertRaisesRegex() does a substring match (if it's like assertRegex()), while we need a full string match to ensure that we don't get undesired prefixes or suffixes.

Reply:
Do you mean:

```python
try:
    # run actual test
except Exception as e:
    self.assertEqual(str(e), error_message)
```

I would argue that using `with assertRaisesRegex()` is better because:

1. It just looks better (e.g. easier to read, I know what to expect)
2. The `.` are only present without wildcards, and in the middle of the message. So, I thought that wouldn't be much of a problem. Let me know if you'd rather have me escape them.